tests/models/layers/test_flash_attn.py

```diff
@@ -6,7 +6,8 @@
 import pytest
 import torch
 
-from llmfoundry.models.layers.attention import (flash_attn_fn,
+from llmfoundry.models.layers import attention
```

Comment:
nit: can we remove this import? The below import should take care of everything

Reply:
We use `attention.attn_bias_shape` later in the same file.  For that we need to explicitly import attention, because `from llmfoundry.models.layers.attention import (flash_attn_fn, ...` does not import attention:

![image](https://github.com/mosaicml/llm-foundry/assets/144760128/76b3d987-3307-4c1f-aa50-36ed70a0fdce)

Reply:
you can do:
```
>>> from llmfoundry.models.layers.attention import attn_bias_shape
>>> attn_bias_shape
<function attn_bias_shape at 0x169318900>
>>> 
```